src/scripts/utils/utils.js

```diff
@@ -0,0 +1,63 @@
+// tableSort/utils.js
+export function convertToNumber(input) {
+  let s = String(input)
+    .trim()
+    .replace(/[^\d,.\-\s\u00A0\u2007\u202F]/g, '')
+    .replace(/[\s\u00A0\u2007\u202F]/g, '');
+
+  s = s.replace(/,/g, '');
+
+  s = s.replace(/-/g, '');
```

Comment:
```suggestion
 s = s.replace(/[,-]/g, '');
```
It's possible to merge it into one method call.